### PR TITLE
2491: Make paddings and icons in poi details consistent

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

This unifies the used paddings in the different poi detail sections.

### Proposed changes

- Content of sections does not have horizontal padding anymore per default (exception: right padding for opening hours to align with header `geöffnet`)
- Icon sizes unified (address marker size increased, external link size decreased)
- Refactoring: Collapsible was using the props wrong way round (initial collapsed made the content NOT being collapsed) and extended to make it more customizable
- Introduced style helper function for adaptiveFontSize

### Side effects

- Usage of Collapsible changed

### Resolved issues

Fixes: #2491.

---

### CODE REVIEW

File: web/src/components/Collapsible.tsx [hunk 1]
```diff
@@ -1,71 +1,54 @@
...
   return (
-    <>
+    <div>
       <CollapsibleHeader
         role='button'
         onClick={() => setCollapsed(!collapsed)}
         tabIndex={0}
-        onKeyPress={() => setCollapsed(!collapsed)}>
-        <Title>{title}</Title>
+        onKeyUp={() => setCollapsed(!collapsed)}>
```
Comment (steffenkleinle) [inline web/src/components/Collapsible.tsx:45]: This was deprecated

**Review by LeandraH — APPROVED** [review]
Nice! Tested in Firefox

**Review by f1sh1918 — APPROVED** [review]
Looks great

I just think if the collapsible is closed it could be centered within the dividers

image

**Comment by steffenkleinle** [comment]
> Looks great
> 
> I just think if the collapsible is closed it could be centered within the dividers

@f1sh1918 do you want to check again if you like the solution?

**Comment by f1sh1918** [comment]
> > Looks great
> > I just think if the collapsible is closed it could be centered within the dividers
> 
> @f1sh1918 do you want to check again if you like the solution?

looks super nice. You can merge @steffenkleinle

**Pull request merged**